Encapsulation/Valeriya/Money.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Encapsulation.Valeriya
+{
+    class Money
+    {
+        private int kopeiky;
+        public int Hrivnas
+        {
+            get
+            {
+                return kopeiky / 100;
+            }
+        }
+        public int Kopeyki
+        {
+            get
+            {
+                return kopeiky % 100;
+            }
+
+        }
+        public Money(int hrivnas, int kopeiky)
+        {
+            this.kopeiky = hrivnas * 100 + kopeiky;
+        }
+
+        public Money(int kopeiky)
+        {
+            this.kopeiky = kopeiky;
+        }
+        public Money(): this(0)
+        {
+        }
+
+        public static Money operator+ (Money operand1, Money operand2)
+        {
+            return new Money(operand1.kopeiky + operand2.kopeiky);
+        }
+
+        public static Money operator* (Money operand1, int operand2)
+        {
+            return new Money(operand1.kopeiky * operand2);
+        }
+
+        public static Money operator* (int operand1, Money operand2)
+        {
+            return new Money(operand2.kopeiky * operand1);
+        }
+
+        public static Money operator- (Money operand1, Money operand2)
+        {
+            return new Money(operand1.kopeiky - operand2.kopeiky);
+        }
+
+        public static Money operator/ (int operand1, Money operand2)
+        {
+            return new Money(operand2.kopeiky / operand1);
+        }
+
+        public static bool operator!= (Money operand1, Money operand2)
+        {
+            bool result = false;
+            if (operand1.kopeiky != operand2.kopeiky)
+            {
+                result = true;
+            }
+
+            return result;
+        }
+
+        public static bool operator== (Money operand1, Money operand2)
+        {
+            bool result = false;
+            if (operand1.kopeiky == operand2.kopeiky)
+            {
+                result = true;
+            }
+
+            return result;
+        }
+
+        public static bool operator> (Money operand1, Money operand2)
+        {
+            bool result = false;
```

Comment:
one-line method is better in this case:
return operand1.kopeiky > operand2.kopeiky;